rdr_service/genomic/genomic_biobank_manifest_handler.py

```diff
@@ -170,7 +170,8 @@ def _get_output_manifest_file_name(genomic_set_id, timestamp=None, cohort_id=Non
     now_cdt_str = (
         _UTC.localize(file_timestamp).astimezone(_US_CENTRAL).replace(tzinfo=None).strftime(OUTPUT_CSV_TIME_FORMAT)
     )
-    cohort = f"-{cohort_id}" if cohort_id else ""
+    cohort = f"_{cohort_id}" if cohort_id else ""
     folder_name = config.getSetting(GENOMIC_BIOBANK_MANIFEST_FOLDER_NAME)
-    full_name = f'{folder_name}/{_MANIFEST_FILE_NAME_PREFIX}{cohort}-{str(genomic_set_id)}-v1{now_cdt_str}.CSV'
+    #full_name = f'{folder_name}/{_MANIFEST_FILE_NAME_PREFIX}{cohort}-{str(genomic_set_id)}-v1{now_cdt_str}.CSV'
```

Comment:
Is the format of the full_name value nailed down or are we leaving both formats in the code just in case?

Reply:
Yes, this can probably be removed.